tensorflow/core/ops/cudnn_rnn_ops_test.cc

```diff
@@ -71,15 +71,15 @@ TEST(CudnnRNNOpsTest, ForwardLstm_ShapeFn) {
 }
 
 TEST(CudnnRNNOpsTest, ForwardV2Lstm_ShapeFn) {
```

Comment:
We need a shape test for V3 as well?

Reply:
Could you please provide an example of this for V3? I don't think I understand how to use and design this test cases.

Reply:
V3 test is for the most part the same as v1/v2, only that v3 has a new param `seq_lengths`, being a vector.